protocols/perf/src/bin/perf.rs

```diff
@@ -198,25 +198,22 @@ async fn custom(server_address: Multiaddr, params: RunParams) -> Result<()> {
     info!("start benchmark: custom");
     let mut swarm = swarm().await?;
 
-    let (server_peer_id, connection_established) =
-        connect(&mut swarm, server_address.clone()).await?;
+    let start = Instant::now();
+
+    let (server_peer_id, _) = connect(&mut swarm, server_address.clone()).await?;
```

Comment:
Why not also refactor `connect` to not run the timers anymore?

Reply:
:+1: done in https://github.com/libp2p/rust-libp2p/pull/4105/commits/ec8f513046e1eac6ab8a6872e755f1a920195270.